WalletWasabi.Gui/Rpc/WasabiJsonRpcService.cs

```diff
@@ -197,7 +197,8 @@ public object[] GetAllKeys()
 				label = x.Label.ToString(),
 				p2wpkhScript = x.P2wpkhScript.ToString(),
 				pubkey = x.PubKey.ToString(),
-				pubKeyHash = x.PubKeyHash.ToString()
+				pubKeyHash = x.PubKeyHash.ToString(),
+				address = x.GetP2wpkhAddress(Global.Network).ToString()
```

Comment:
I think you don't need the `.ToString()`

Reply:
I don't know, it was used here:
https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Gui/Rpc/WasabiJsonRpcService.cs#L41
https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Gui/Rpc/WasabiJsonRpcService.cs#L85

Reply:
Let's keep it there. Makes it symmetric :)